addon/index.ts

```diff
@@ -9,8 +12,16 @@ class SparklesComponent<T = object> {
   // didRender() {}
   destroy() {}
 }
-setComponentManager('sparkles', SparklesComponent);
+
+if (gte('3.8.0')) {
```

Comment:
Should be `gte(‘3.8.0-beta.1’)` otherwise folks in the beta will get the deprecation still